tests/unit/all.js

```diff
@@ -249,14 +249,14 @@ describe('code snippet example', () => {
     const limit = 2;
     it(`_.split("${source}", "${separator}")`, () => {
       assert.equal(
-        _.split(source, separator),
-        source.split(separator)
+        JSON.stringify(_.split(source, separator)),
```

Comment:
use `deepEqual` instead of `JSON.stringify`

Reply:
Good ideia. I updated and commit, thank's.